packages/eslint-plugin-ui5-define-jsdocs/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@ui5/eslint-plugin-ui5-define-jsdocs",
+  "version": "0.0.2",
+  "description": "ui5 apps validations",
+  "keywords": [
+    "eslint",
+    "eslintplugin",
+    "eslint-plugin"
+  ],
+  "main": "lib/index.js",
+  "scripts": {
+    "test": "nyc mocha tests --recursive"
+  },
+  "dependencies": {
+    "doctrine": "^3.0.0",
+    "eslint": "^5.16.0",
+    "lodash": "^4.17.11",
+    "requireindex": "^1.2.0",
+    "string-format": "^2.0.0"
+  },
+  "devDependencies": {
```

Comment:
1. formatting is handled by root package. 
2. we use prettier for formatting in this repo (not standard)

Reply:
we use string-format for messages, we think it should stay